### PR TITLE
Fix main and "More" menus and add prompt to modules for unsupported agents

### PR DESCRIPTION
**Pull request by Desvelao**
Hi team, this PR resolves:
  - Fixed main menu when selected an unsupported agent
  - Fixed More menu in agent view for an unsupported agent
  - Added prompt to show when the module is not supported by the agent
  - Created prompt component to show the module is not supported by the agent
  - Created HOC for the not supported module
  
  Closes: #2836
  
  Screenshots:
  Prompt in modules:
  image

### CODE REVIEW

File: public/components/wz-menu/wz-menu-agent.js [hunk 1]
```diff
@@ -44,32 +44,54 @@ class WzMenuAgent extends Component {
...
+      vuls: { id: WAZUH_MODULES_ID.VULNERABILITIES, text: 'Vulnerabilities' },
+      virustotal: { id: WAZUH_MODULES_ID.VIRUSTOTAL, text: 'VirusTotal' },
+      osquery: { id: WAZUH_MODULES_ID.OSQUERY, text: 'Osquery' },
+      docker: { id: WAZUH_MODULES_ID.DOCKER, text: 'Docker Listener' },
+      mitre: { id: WAZUH_MODULES_ID.MITRE_ATTACK, text: 'MITRE ATT&CK' },
+      pci: { id: WAZUH_MODULES_ID.PCI_DSS, text: 'PCI DSS' },
+      gdpr: { id: WAZUH_MODULES_ID.GDPR, text: 'GDPR' },
+      hipaa: { id: WAZUH_MODULES_ID.HIPAA, text: 'HIPAA' },
+      nist: { id: WAZUH_MODULES_ID.NIST_800_53, text: 'NIST 800-53' },
+      tsc: { id: WAZUH_MODULES_ID.TSC, text: 'TSC' }
```
Comment (gabiwassan) [inline public/components/wz-menu/wz-menu-agent.js:65]: Great job with all of these constants! 👏🏼 
Maybe we can create constants for the `text` values.

**Review by gabiwassan — APPROVED** [review]
LGTM! 🚀

**Review by asteriscos — APPROVED** [review]
LGTM, but I would consider changing the prompt message to _"Module not supported by the agent"_ 
Great job!

File: public/components/wz-menu/wz-menu-agent.js (hunk 1 above)
Comment (Desvelao) [reply public/components/wz-menu/wz-menu-agent.js:65]: Yes, you are right, we should move them to constants too. These `text` values are using in more files, we should refactor them in the next patches.

**Comment by Desvelao** [comment]
> LGTM, but I would consider changing the prompt message to _"Module not supported by the agent"_
> Great job!

I replaced the prompt message. 👍

**Pull request merged**